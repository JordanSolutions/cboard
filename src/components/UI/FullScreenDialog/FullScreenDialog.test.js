import React from 'react';
import { shallowMatchSnapshot } from '../../../common/test_utils';
import FullScreenDialog from './FullScreenDialog';


describe('FullScreenDialog tests', () => {
  test('default renderer', () => {
    shallowMatchSnapshot(<FullScreenDialog />);
  });
});

