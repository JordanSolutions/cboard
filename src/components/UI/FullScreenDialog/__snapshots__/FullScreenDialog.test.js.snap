// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FullScreenDialog tests default renderer 1`] = `
<FullScreenDialog
  classes={
    Object {
      "appBar": "FullScreenDialog-appBar-1",
      "container": "FullScreenDialog-container-3",
      "content": "FullScreenDialog-content-4",
      "title": "FullScreenDialog-title-2",
    }
  }
  onClose={[Function]}
/>
`;
